app/src/main/java/com/kickstarter/libs/Environment.java

```diff
@@ -35,6 +35,7 @@
   public abstract Koala koala();
   public abstract KSCurrency ksCurrency();
   public abstract KSString ksString();
+  public abstract Koala lake();
```

Comment:
Why is the `lake` property of type `Koala`? Is it to do exactly the same but with "clean properties"? 🤔

Reply:
`Koala` is just a wrapper around a `TrackingClient`. And yeah, `lake` can track properties but with the different JSON structure and "clean properties"